File: src/domain/models/escola.py
from typing import Optional


class Escola:
    def __init__(self, nome: str, id_municipio: int, id: int, nome_usuario: Optional[str] = None, senha: Optional[str] = None):
        self.id = id
        self.nome = nome
        self.id_municipio = id_municipio
        self.nome_usuario = nome_usuario
        self.senha = senha

File: frontend/entites/Escola.ts
/* eslint-disable @typescript-eslint/no-explicit-any */
import { Escola } from "@/types/Escola";

export const EscolaAPI = ({ nome, idMunicipio, nomeUsuario, senha }: any) => ({
  nome: nome,
  id_municipio: idMunicipio,
  nome_usuario: nomeUsuario,
  senha: senha,
});

export const EscolaInfo = (data: any): Escola => ({
  id: data.id,
  nome: data.nome,
  idMunicipio: data.id_municipio,
  nomeUsuario: data.nome_usuario,
  senha: data.senha,
});
